<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Overlook</class>
 <widget class="QMainWindow" name="Overlook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EtheriaDit</string>
  </property>
  <widget class="QWidget" name="c_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="y_layout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QMdiArea" name="mdi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="b_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="m_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="a_load"/>
    <addaction name="a_save"/>
    <addaction name="a_close"/>
   </widget>
   <addaction name="m_file"/>
  </widget>
  <widget class="QStatusBar" name="s_bar"/>
  <action name="a_load">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="a_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="a_close">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
